.github/workflows/wheels-manylinux-build.yml

```diff
@@ -0,0 +1,201 @@
+
+      # Extra repository that will be cloned into the project directory.
+      extra-repo:
+        required: false
+        type: string
+        default: ''
+      extra-repo-sha:
+        required: false
+        type: string
+        default: ''
+      # Note that this is the _name_ of a secret containing the key, not the key itself.
+      extra-repo-deploy-key: 
+        required: false
+        type: string
+        default: ''
+
+      # versioneer and twine settings
+      python-package-versioneer-override:
+        required: false
+        type: string
+        default: ''
+
+      cibw-environment:
+        type: string
+        required: false
+        default: ''
+      cibw-before-all-amd64:
+        required: false
+        type: string
+        default: 'true'
+      cibw-before-all-arm64:
+        required: false
+        type: string
+        default: 'true'
+      cibw-before-build:
+        required: false
+        type: string
+        default: 'true'
+      post-repair-amd64:
+        required: false
+        type: string
+        default: 'true'
+      post-repair-arm64:
+        required: false
+        type: string
+        default: 'true'
+
+      skbuild-configure-options:
+        required: false
+        type: string
+        default: ''
+      skbuild-build-options:
+        required: false
+        type: string
+        default: ''
+
+jobs:
+  wheel-epoch-timestamp:
+    name: wheel epoch timestamper
+    runs-on: ubuntu-latest
+    outputs:
+      rapids_epoch_timestamp: ${{ steps.get_epoch_time.outputs.rapids_epoch_timestamp }}
+
+    steps:
+    - name: Get epoch timestamp to use as nightly version
+      id: get_epoch_time
+      run: echo "RAPIDS_EPOCH_TIMESTAMP=$(date +%s)" >> "${GITHUB_OUTPUT}"
+
+  wheel-build:
+    name: cibuildwheel ${{ matrix.arch }} ${{ matrix.python }} ${{ matrix.ctk }}
+    needs: wheel-epoch-timestamp
+    strategy:
+      matrix:
+        python: ['3.8', '3.10']
+        arch: [amd64, arm64]
+        ctk: ['11.8.0', '12.0.0']
+    runs-on:
+      - self-hosted
+      - linux
+      - ${{ matrix.arch }}
+      - cpu16
+    container:
+      # ctk version of the cibw container is irrelevant in the manylinux case
+      # it's simply a launcher for cibuildwheel
+      image: "rapidsai/cibuildwheel:cuda-runtime-11.8.0-ubuntu20.04"
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock
+      env:
+        RAPIDS_BUILD_TYPE: ${{ inputs.build_type }}
+        RAPIDS_PY_VERSION: ${{ matrix.python }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.RAPIDSAI_GHA_AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.RAPIDSAI_GHA_AWS_SECRET_ACCESS_KEY }}
+
+    steps:
+    - name: checkout code repo
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.repo }}
+        ref: ${{ inputs.sha }}
+        fetch-depth: 0 # unshallow fetch for setuptools-scm
+        persist-credentials: false
+
+    - name: Standardize repository information
```

Comment:
This step seems identical in every wheel workflow, and it's also present in the [conda workflows](https://github.com/rapidsai/shared-action-workflows/blob/main/.github/workflows/conda-upload-packages.yaml). Can we extract it into a composite action or a reusable workflow (I don't recall off the top of my head which -- if either -- would be suitable for things like the `GITHUB_ENV`/`GITHUB_OUTPUT` modifications done in this step).

Reply:
Yeah I'll see if that's possible (without creating some unwanted behavior such as the modification of GITHUB_* depending on whether the step is in a composite action vs. embedded in a reusable workflow).

Reply:
Seems to have worked fine